### PR TITLE
Site Importer: Show siteUrl in the preview step

### PR DESCRIPTION
**Pull request by jblz**
#### Changes proposed in this Pull Request

* Include the source site's URL in the "Is this your site?" / "Are you sure" screen.
* styling

#### Testing instructions

1. Visit the importers section of a site (`/settings/import/{site-slug}`)
1. Choose the Wix importer
1. Enter a valid url to import and click "Continue" (example: https://hi6822.wixsite.com/eat-here-its-good)
1. You should see the URL to import above the preview image
1. Clicking on the URL opens the link in a new tab/window

screen shot 2018-10-29 at 15 52 08

### CODE REVIEW

**Comment by creativecoder** [comment]
Pushed work in progress. Still to do: needs some additional styling work for smaller screen sizes.

File: client/my-sites/importer/style.scss [hunk 1]
```diff
@@ -219,56 +219,69 @@
 	& > div {
 		margin-bottom: 10px;
 	}
+}
 
-	.site-importer__site-importer-url-input {
```
Comment (creativecoder) [inline client/my-sites/importer/style.scss]: Un-nesting selectors that don't need to be nested.

File: client/my-sites/importer/style.scss [hunk 2]
```diff
@@ -219,56 +219,69 @@
...
-		button {
-			float: none;
-			flex-grow: 0;
-			flex-shrink: 0;
+.site-importer__site-importer-confirm-site-pane-container {
+	display: flex;
+	align-content: center;
+	align-items: flex-start;
+	// justify-content: center;
+	flex-wrap: wrap;
```
Comment (creativecoder) [inline client/my-sites/importer/style.scss]: New style

File: client/my-sites/importer/style.scss [hunk 3]
```diff
@@ -219,56 +219,69 @@
...
+		flex-grow: 0;
+		flex-shrink: 0;
+
+		&:first-of-type {
+			margin-left: 0;
 		}
 	}
 }
 
+.site-importer__site-importer-confirm-site-label {
```
Comment (creativecoder) [inline client/my-sites/importer/style.scss]: New styles

File: client/my-sites/importer/style.scss [hunk 4]
```diff
@@ -219,56 +219,69 @@
...
+	flex-grow: 1;
+	display: flex;
+	flex-direction: column;
+	margin-right: 10px;
```
Comment (creativecoder) [inline client/my-sites/importer/style.scss]: New style

File: client/my-sites/importer/style.scss [hunk 5]
```diff
@@ -219,56 +219,69 @@
...
+	// justify-content: center;
+}
+
+.site-importer__source-url {
```
Comment (creativecoder) [inline client/my-sites/importer/style.scss]: New styles

**Comment by creativecoder** [comment]
Ready for review

**Comment by jblz** [comment]
Github won't let me "approve" this b/c it's my own PR, but this is working well for me.  🚢

**Comment by creativecoder** [comment]
Awesome, I'll "review", merge, and deploy.

**Comment by spen** [comment]
FWIW - I tested earlier and it looks good :) 🎉 (Just forgot to hit accept) Tested at various breakpoints - all work well.

**Pull request merged**